CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.0-rc.7 (2018-12-13)
+
+- Added option to take action before modals are closed and prevent them from closing. [#7](https://github.com/blackbaud/skyux-modals/pull/7)
+- Fixed section forms in modals always being 460px. [#18](https://github.com/blackbaud/skyux-modals/pull/18)
```

Comment:
Seems like we should specify that we are referring to height (as opposed to width). Also, I think "always being 460px" could be misread to indicate that we made them always 460px. What do you think of tightening up to just:

"Fixed section forms in modals to set minimum height of 460px."

Reply:
That sounds good :+1: